test/uk/gov/hmrc/pillar2externalteststub/controllers/ObligationsAndSubmissionsControllerSpec.scala

```diff
@@ -234,7 +234,7 @@ class ObligationsAndSubmissionsControllerSpec
           canAmendCheck(boundaryDate.minusDays(1), expectedStatus = false)
         }
 
-        "true when current date is within 12 months from the dueDate" in {
+        "true when current date is within 12 months from the dueDate" ignore {
```

Comment:
```suggestion
        "true when current date is within 12 months from the dueDate" in {
```

Why is the test being ignored?

Reply:
@gianick https://jira.tools.tax.service.gov.uk/browse/PIL-2420